src/controlCenter/widgets/volume/pulseDevice.vala

```diff
@@ -0,0 +1,134 @@
+using PulseAudio;
```

Comment:
We should add a perma link to the SwaySettings file as a comment at the top. Something like `// From SwaySettings PulseAudio page: PERMALINK`